jsconfig.json

```diff
@@ -5,7 +5,7 @@
 		"target": "ES6",
 		"baseUrl": ".",
 		"paths": {
-			"*": [ "test/*", "server/*", "client/*" ]
+			"*": [ "test/*", "client/*" ]
```

Comment:
Can't `server` paths still be accessed like a root?

```suggestion
			"*": [ "test/*", "client/*", "client/server/*" ]
```

Reply:
@sirreal I don't think so? What makes you think they still can be?

Reply:
I wasn't aware this had changed, seems like you're right 👍

Reply:
Thanks for checking and confirming 👍